argocon-demos/2024-mastering-argo-workflows-at-scale/README.md

```diff
@@ -0,0 +1,18 @@
+# Mastering Argo Workflows at Scale: A Practical Guide to Scalability Excellence
+
+[![Pipekit Logo](https://raw.githubusercontent.com/pipekit/talk-demos/main/assets/images/pipekit-logo.png)](https://pipekit.io?utm_campaign=talk-demos)
+
+## The talk
+<!-- The talk recording [can be found here](https://youtu.be/wGik7O1qZbE). -->
+
+The slide deck for this talk can be found [here](assets/slide-deck.pdf).
+
+
+
+---
+
+## About Pipekit
+
+Pipekit is the control plane for Argo Workflows. Platform teams use Pipekit to manage data & CI pipelines at scale, while giving developers self-serve access to Argo. Pipekit's unified logging view, enterprise-grade RBAC, and multi-cluster management capabilities lower maintenance costs for platform teams while delivering a superior devex for Argo users. Sign up for a 30-day free trial at [pipekit.io/signup](https://pipekit.io/signup?utm_campaign=talk-demos).
```

Comment:
we should probably add in workflow metrics/ observability now! 

should we call it "workflow metrics dashboard" or "workflow observability dashboard"?

> Pipekit's unified logging view, workflow metrics dashboard, enterprise-grade RBAC, and multi-cluster management capabilities lower maintenance costs for platform teams while delivering a superior devex for Argo users.

> Pipekit's unified logging view, workflow observability dashboard, enterprise-grade RBAC, and multi-cluster management capabilities lower maintenance costs for platform teams while delivering a superior devex for Argo users.

I kind of like "metrics" instead of "observability" to lessen the jargon load on the sentence, but curious what ideas you might have.